src/modules/RA/DataLayer/components/DataLayerTabbedForm.js

```diff
@@ -46,7 +46,8 @@ const styles = {
   },
 };
 
-const CustomFormTab = ({ hidden, ...props }) => (
+// Best performance when tabs have heavy content and produce multiple render
+const LazyFormTab = ({ hidden, ...props }) => (
   hidden
     ? null
     : <FormTab hidden={hidden} {...props} />
```

Comment:
If hidden is a boolean value, you can one line the return. And the hidden prop would be useless
`hidden && <FormTab {...props} />`

Reply:
Finally, it wouldn't work for the first suggestion, but it could be still relevant for the secnd part:

```jsx
const LazyFormTab = ({ hidden, ...props }) => (
  hidden
    ? null
    : <FormTab {...props} />
```

Reply:
I get this error with your code : `Invariant Violation: LazyFormTab(...): Nothing was returned from render`

Reply:
Ok for the second part !

Reply:
Done